guide_drafts/trait_derives.md

```diff
@@ -52,10 +61,11 @@ That means we'll be able to use `expect()` to handle our error conditions.
 
 ### Example
 
-*Note: src/lib.rs with #[macro_use] extern crate diesel; not shown*
-
-*src/models.rs*
 ```rust
+// File: src/models.rs
+
+use schema::users; // Brings table! into scope
```

Comment:
Do you think this comment is misleading? Should it say "brings the user table into scope"?